.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Describe the change
+
+Explanation of what this PR does
+
```

Comment:
```suggestion
### Automation testing 

If applicable, explain the case scencarios covered by **unit / integration / e2e** tests created for this PR.

```

With all the feedback received I think most of us agree to have some kind of testing section (I don't see a PR checklist just to check if unit/e2e tests have been created). Brief explanation of the test case scenarios can be useful to reviewer (and qa) to determine if the PR is well covered by tests or not.

Reply:
>  Brief explanation of the test case scenarios
+1

About testing strategy: we usually don't have it, or late in the development cycle. Is there such a thing like template for epic, or epic checklist?

Reply:
Currently there is no such epic template. Since there are very few epics I am not sure we should have a template just for epics. If you see testing strategy is missing in one of the epics, just contact the epic owner to discuss about it.